var/spack/repos/builtin/packages/vtk-m/package.py

```diff
@@ -18,10 +18,11 @@ class VtkM(CMakePackage, CudaPackage):
     architectures."""
 
     homepage = "https://m.vtk.org/"
-    url      = "https://gitlab.kitware.com/api/v4/projects/vtk%2Fvtk-m/repository/archive.tar.gz?sha=v1.3.0"
+    url      = "https://gitlab.kitware.com/api/v4/projects/vtk%2Fvtk-m/repository/archive.tar.gz?sha=v1.5.0"
```

Comment:
Rather than use the "miscellaneous sha to archive" download API, can we just switch the URLs to the release download?  i.e. `https://gitlab.kitware.com/vtk/vtk-m/-/archive/v1.5.0/vtk-m-v1.5.0.tar.gz`

Reply:
@chuckatkins Updated url, and updated the SHA1 where they differ from the new source.